### PR TITLE
adding payload for subscriptions webhook url

### PR DESCRIPTION
**Pull request by jayachandhar**

### CODE REVIEW

File: src/approver/paths/approver@subscriptions.yaml [hunk 1]
```diff
@@ -33,3 +33,29 @@ post:
         application/json:
           schema:
             $ref: '../../components/schemas/401.yaml'
+  callbacks:
+    'REPORT.*':
```
Comment (jayachandhar) [inline src/approver/paths/approver@subscriptions.yaml:37]: event_name

File: reference/approver.yaml [hunk 2]
```diff
@@ -522,3 +522,38 @@ paths:
...
+                      additionalProperties: false
+                      properties:
+                        resource:
+                          type: string
+                          example: REPORT
+                        action:
+                          type: string
+                          enum:
+                            - DRAFT
+                            - APPROVER_PENDING
```
Comment (aditya999123) [inline reference/approver.yaml:543]: action and state are different things
should fetch from class actionenum

File: src/fyler/paths/fyler@subscriptions.yaml [hunk 3]
```diff
@@ -33,3 +33,29 @@ post:
...
+                schema:
+                  type: object
+                  additionalProperties: false
+                  properties:
+                    resource:
+                      type: string
+                      example: 'REPORT'
+                    action:
+                      type: string
+                      enum: [ 'DRAFT', 'APPROVER_PENDING', 'APPROVED', 'APPROVER_INQUIRY','PAYMENT_PENDING', 'PAYMENT_PROCESSING', 'PAID' ]
```
Comment (aditya999123) [inline src/fyler/paths/fyler@subscriptions.yaml:52]: action and state are different things
should fetch from class actionenum

File: src/fyler/paths/fyler@subscriptions.yaml [hunk 4]
```diff
@@ -33,3 +33,29 @@ post:
...
+                      example: 'APPROVED'
+                    data:
+                      $ref: '../../components/schemas/report.yaml#/report_out'
```
Comment (aditya999123) [inline src/fyler/paths/fyler@subscriptions.yaml:55]: this is not same as report_out
updated_by_user is an added field

File: src/approver/paths/approver@subscriptions.yaml [hunk 5]
```diff
@@ -33,3 +33,29 @@ post:
...
+      '{$request.body#/data/webhook_url}':
+        post:
+          requestBody:
```
Comment (aditya999123) [inline src/approver/paths/approver@subscriptions.yaml:40]: can we use some reference, this is essentially same as fyler

**Pull request merged**